frontend/src/views/PackagePopularity.vue

```diff
@@ -1,5 +1,167 @@
-<script setup lang="ts"></script>
+<script setup lang="ts">
+import PackagePopularityElement from '../components/PackagePopularityElement.vue'
 
-<template>PACKAGE POPULARITY</template>
+import { usePackagePopularityStore } from '../stores/packagePopularity'
+const packagePopularityStore = usePackagePopularityStore()
+import { useDisplay } from 'vuetify'
+const { lgAndUp } = useDisplay()
+import { computed } from 'vue'
+import VueApexCharts from 'vue3-apexcharts'
 
-<style scoped></style>
+const chartOptions = computed(() => {
+  return {
+    chart: { type: 'bar', height: '600px', toolbar: { show: false } },
+    plotOptions: {
+      bar: {
+        horizontal: true,
+        distributed: true,
+      },
+    },
+    dataLabels: {
+      textAnchor: 'start',
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      formatter: function (_: any, opt: any) {
+        return packagePopularityStore.kpiValue.items[opt.dataPointIndex].package
+      },
+      style: {
+        colors: ['#000'],
+      },
+    },
+    xaxis: {
+      labels: {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        formatter: function (val: any, _: any) {
+          return val + ' %'
```

Comment:
```suggestion
          return `${val} %`
```
Wouldn't it be better to use backtick strings instead? I find it more explicit and in-line with the f-strings we use on the backend